lib/index.js

```diff
@@ -80,6 +86,15 @@ Optimizely.prototype.track = function(track) {
   var revenue = track.revenue();
   if (revenue) payload.revenue = revenue *= 100;
   push(['trackEvent', track.event(), payload]);
+
+  var optimizelyClientInstance = window.optimizelyClientInstance;
+  if (optimizelyClientInstance && optimizelyClientInstance.track) {
+    var attributes = track.properties();
+    var userId = attributes.userId || track.userId();
```

Comment:
@hankim813 The Fullstack JS SDK is not loaded synchronously. It is bundled into the developer's code via something like `Webpack` or `Browserify`. I am fine with leaving the integration here as our `track` API should not change much going forward. 

Let's follow up on email for a screenshare so I can repro with you around.